scripts/tests/py/TC_RR_1_1_Test.py

```diff
@@ -0,0 +1,880 @@
+#
+#    Copyright (c) 2022 Project CHIP Authors
```

Comment:
this should not be a copy of the full TC_RR_1_1. We want to focus on the content of run arguments.

this should be a 5-10 line file (i doubt we even need copyright blurb for test data) and not be a full 100s of lines file.

please delete all lines after line 24 and you can remove the license blurb too. This should be a minimal file, the same as we would have for test data.

We could even use make our parsers so that we can pass in raw strings (so tests have the strings in them)

Reply:
Naming should also not be `TC_RR_....`. It should be something that explains the test, like `simple_run_args.py` or `simple_run_args.txt` or similar.